canonical-web-and-design/juju.is
# About page

## Done
**Don't merge** this PR currently includes placeholder images, and the copy doc is still being finalised

Added /about page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041/about or visit the [demo](https://juju-is-194.demos.haus/about)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it matches the [design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/6079637868f7b589d49d8259) and the [copy doc](https://docs.google.com/document/d/186zQ7bYE-jXqNNTyKEvSzjq-yP-MiMW2zHKbgyrBrm4/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2006
